components/ws-manager/pkg/manager/create.go

```diff
@@ -577,6 +590,28 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 			gitpodGUID := int64(133332)
 			pod.Spec.SecurityContext.FSGroup = &gitpodGUID
 
+		case api.WorkspaceFeatureFlag_PROTECTED_SECRETS:
+			for _, c := range pod.Spec.Containers {
+				if c.Name != "workspace" {
```

Comment:
is this check necessary? this implies we have more then one container in the spec. Do we now?

Reply:
That's right - today we don't have multiple container, and it's not that likely we will in the future (we have had multiple containers in the past).

I'd be ok with removing the check, and equally with leaving it in.
IMHO it's a tad more complete, because the env vars of other containers (should we have them) would likely not be part of the secret.